client/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "client",
```

Comment:
What do you think about this being something like `Social Share client` or whatever but just client